packages/studio/package.json

```diff
@@ -13,7 +13,8 @@
     "lint": "prettier --check . && eslint .",
     "format": "prettier --write .",
     "cleanup": "rimraf node_modules .svelte-kit dist",
-    "test": "vitest"
+    "test": "vitest run",
```

Comment:
oh, interesting, does the default `vitest` command start a watcher vs the `run` running once?

Reply:
Correct 👍